packages/ndla-video-search/src/VideoTabs.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2016-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Tabs from 'ndla-tabs';
+
+class VideoTabs extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      index: 0,
+    };
+    this.handleOnSelect = this.handleOnSelect.bind(this);
+  }
+
+  handleOnSelect(index, last) {
+    const { onSearchTypeChange, tabs } = this.props;
+    if (index !== last) {
+      this.setState({ index });
+      onSearchTypeChange(tabs[index].title.toLowerCase());
+    }
+  }
+
+  render() {
+    const { index } = this.state;
+    const { tabs } = this.props;
+
+    return (
+      <Tabs selectedIndex={index} onSelect={this.handleOnSelect} tabs={tabs} />
```

Comment:
Her skulle du ikke trenge å ha noe egen index state. Bare `<Tabs onSelect={this.handleOnSelect} tabs={tabs} />`